src/mono/mono.proj

```diff
@@ -470,6 +470,12 @@
       <_MonoAOTCFLAGS Include="-Wl,--build-id=sha1" />
       <_MonoAOTCXXFLAGS Include="-Wl,--build-id=sha1" />
     </ItemGroup>
+    <ItemGroup Condition="'$(TargetsLinuxMusl)' == 'true'">
+      <_MonoAOTCFLAGS Include="-lucontext" />
+      <_MonoAOTCXXFLAGS Include="-lucontext" />
+      <_MonoCFLAGS Include="-lucontext" />
+      <_MonoCXXFLAGS Include="-lucontext" />
+    </ItemGroup>
```

Comment:
The following is due to `ucontext` symbols being provided by a seperate `libucontext` package, as `musl` does not natively support it.